packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -99,6 +99,8 @@ registerFragment(`
     nominationCount2018
     reviewCount2018
 
+    tagRelevance
```

Comment:
Does this really need to be in the `PostsBase` fragment? It's a pretty expensive field.

Reply:
So on one hand, this should certainly get a custom fragment rather than using PostsBase, which I think basically only fetches tagRelevance.

On the other hand, somewhat surprised that tagRelevance shouldn't be included in PostsBase – it's pretty small AFAICT. (This is more of question about the general principle, agree that this PR shouldn't add it here)